File: content/realm/examples/react-native/legacy/__tests__/js/Models/Car.js
import Realm from 'realm';

// :snippet-start: js-car-schema
class Car extends Realm.Object {
  static schema = {
    name: 'Car',
    properties: {
      make: 'string',
      model: 'string',
      miles: {type: 'int', default: 0},
      timestamp: {
        type: 'int',
        default: () => Math.round(new Date().getTime() / 1000),
      },
    },
  };
}
// :snippet-end:
export default Car;

File: content/realm/examples/dart/test_sdk_example/test/car.dart
// :snippet-start: define-model-flutter
import 'package:realm/realm.dart';

part 'car.realm.dart';

@RealmModel()
class _Car {
  @PrimaryKey()
  late ObjectId id;

  late String make;
  late String? model;
  late int? miles;
}
// :snippet-end:
